inc/Base/Activate.php

```diff
@@ -53,16 +53,19 @@ private static function create_database() {
 		$charset_collate = $wpdb->get_charset_collate();
 		$smaily     = "CREATE TABLE $table_name (
 				id int(11) NOT NULL AUTO_INCREMENT,
-				enable tinyint(1) DEFAULT NULL,
+				enable tinyint(1) DEFAULT 0,
 				subdomain varchar(255) DEFAULT NULL,
 				username varchar(255) DEFAULT NULL,
 				password varchar(255) DEFAULT NULL,
 				syncronize_additional varchar(255) DEFAULT NULL,
-				enable_cart tinyint(1) DEFAULT NULL,
+				enable_cart tinyint(1) DEFAULT 0,
 				cart_autoresponder varchar(255) DEFAULT NULL,
 				cart_autoresponder_id int(10) DEFAULT NULL,
 				cart_cutoff int(10) DEFAULT NULL,
 				cart_options varchar(255) DEFAULT NULL,
+				checkbox_display varchar(255) DEFAULT 'visible_unchecked',
+				checkbox_order varchar(255) DEFAULT 'after',
+				checkbox_location varchar(255) DEFAULT 'checkot_billing_form',
```

Comment:
Typo in default value. Should be `checkout_billing_form`.

Reply:
Fixed in https://github.com/sendsmaily/smaily-woocommerce-plugin/pull/45/commits/f6ca3f87e11bae7df8ce19dfd5a39e70d5f144dc